assets/src/components/admin/admin_tables.tsx

```diff
@@ -186,6 +186,19 @@ const alertsColumn = {
   FormCell: FormTextarea,
 };
 
+const includeLogoColumn = {
+  Header: "Include Logo?",
+  accessor: (row) => row.app_params.include_logo_in_header,
+  mutator: (row, value) => {
```

Comment:
I think we should be able to simplify this using `buildAppParamAccessor` and `buildAppParamMutator` here.